using-mobile-extensions/adobe-edge-identity/edge-identity-faq.md

```diff
@@ -125,4 +125,184 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
+
+Each identity extension has its own API to retrieve their respective ECIDs as well. Use [Identity.getExperienceCloudId](./edge-identity-api-reference#getexperiencecloudid) to get the AEP Edge Identity ECID, and [Identity.getExperienceCloudId](../mobile-core/identity/identity-api-reference#getexperiencecloudid) to get the AEP Identity direct solution ECID.
+
+
+## Q: How can I get all the identifiers used by the SDK when using both AEP Edge extensions and Adobe Solutions extensions?
+### A: Use both [getSdkIdentities](../mobile-core/mobile-core-api-reference#retrieving-stored-identifiers) and [getIdentities](./edge-identity-api-reference#getidentities)
+
+To get the identifiers used by the Adobe Solutions extensions, call [getSdkIdentities](../mobile-core/mobile-core-api-reference#retrieving-stored-identifiers).
+
+To get the identifiers used by the AEP Edge extensions, call [getIdentities](./edge-identity-api-reference#getidentities).
+
+
+## Q: How can I clear all the identifiers from the SDK when using both AEP Edge extensions and Adobe Solutions extensions?
+### A: Set privacy status to `optedOut` and call `resetIdentities`
+
+To clear the identifiers used by the Adobe Solutions extensions, call [setPrivacyStatus](../resources/privacy-and-gdpr#setprivacystatus) and set the privacy status to `optedOut`.
+
+To clear the identifiers used by the AEP Edge extensions, call [resetIdentities](../mobile-core/mobile-core-api-reference#reset-identities)
+
+
+## Q: What steps are needed to generate a new Experience Cloud ID (ECID) for a user when using both AEP Edge extensions and Adobe Solutions extensions?
+### A: Both identity extensions' ECID must be regenerated in sequence to avoid linking the old and new ECIDs in Adobe Experience Platform.
+
+When using Real-time Customer Profile and Identity Service, the ECIDs from both identity extensions are linked together in the customer's Identity Graph. Care must be taken when regenerating new ECIDs such that the old and new ECIDs are not linked within the same Identity Graph.
+
+Perform the following API calls to regenerate the ECIDs in sequence:
+1. Set [privacy status](../resources/privacy-and-gdpr#setprivacystatus) to `optedOut` to clear the ECID from the AEP Identity direct service extension.
+2. Call [resetIdentities](./edge-identity-api-reference#resetidentities) to regenerate a new ECID in the AEP Edge Identity extension.
+3. Call [getExperienceCloudId](./edge-identity-api-reference#getexperiencecloudid) on the AEP Edge Identity extension. This ensures the new ECID is generated before continuing.
+4. Set [privacy status](../resources/privacy-and-gdpr#setprivacystatus) to `optedIn` to generate a new ECID in the AEP Identity direct service extension.
```

Comment:
For this question can we provide a sample code with the sequence instead of text?

Reply:
What do you think of having both?  I can add code examples after the list. I like having links to the mentioned APIs.